src/parameter-container.ts

```diff
@@ -22,6 +22,15 @@ export class ParameterContainer {
   addParam(value: any, name?: string): Parameter {
     return this.parameterBag.addParam(value, name);
   }
+  
+  /**
+   * Adds a new parameter to the bag.
+   */
+  addParams(params: Dictionary<any>): Parameter[] {
+    return map(params, (value, name) => {
+      return this.parameterBag.addParam(value, name);
+    });
```

Comment:
Do you think it is useful for this function to return the params here? I think it might be better for this to just return nothing instead. Especially because the returned params are only the ones that are being added and not every param in the bag.

Reply:
@jamesfer yeah, makes sense. I originally did not put it but then added it for consistency with `addParam`.

Reply:
@bboure oh yeah, coming back to this it does make sense to return the params for the same reason as in my other comments. I think it will probably make the most sense to return a `Dictionary<Parameter>` in both cases